src/components/HeaderWithLanguage/LearningpathConnection.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { useEffect, useState } from 'react';
+import styled from '@emotion/styled';
+import { colors } from '@ndla/core';
+import { injectT, tType } from '@ndla/i18n';
+import { LearningPath } from '@ndla/icons/contentType';
+import Modal, { ModalHeader, ModalCloseButton, ModalBody } from '@ndla/modal';
+import Tooltip from '@ndla/tooltip';
+import { normalPaddingCSS } from '../HowTo';
+import ElementList from '../../containers/FormikForm/components/ElementList';
+import { Learningpath } from '../../interfaces';
+import { fetchLearningpathsWithArticle } from '../../modules/learningpath/learningpathApi';
+
+interface Props {
+  id: number;
+}
+
+const LearningpathIcon = styled(LearningPath)`
+  color: ${colors.brand.tertiary};
+
+  &:hover,
+  &:focus {
+    color: ${colors.brand.primary};
+    border: 1 px ${colors.brand.tertiary};
+  }
+`;
+
+const LearningpathConnection = ({ t, id }: Props & tType) => {
+  const [learningpaths, setLearningpaths] = useState<Learningpath[]>([]);
+
+  useEffect(() => {
+    fetchLearningpathsWithArticle(id).then(setLearningpaths);
+  }, [id]);
+
+  return (
+    <Modal
+      backgroundColor="white"
+      narrow
+      wrapperFunctionForButton={(activateButton: any) => (
+        <Tooltip tooltip={t('form.learningpathConnections.sectionTitle')}>{activateButton}</Tooltip>
+      )}
+      activateButton={<LearningpathIcon css={normalPaddingCSS} />}>
```

Comment:
Kun vise knappen når det finnes stier:
```suggestion
      activateButton={learningpaths?.length > 0 && <LearningpathIcon css={normalPaddingCSS} />}>
```

Reply:
Sikkert ikkje den beste måten å gjøre det på, men vil funke.

Reply:
Og om du endrer pointer til å være :point_up_2: så er det lettere å sjå at du kan klikke på den.